bundles/org.openhab.binding.mideaac/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,270 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mideaac"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Thing Type -->
+	<thing-type id="ac">
+		<label>Midea Air Conditioner</label>
+		<description>Midea Air Conditioner with USB WIFI stick. There are 2 versions: v2 - without encryption, v3 - with
+			encryption - Token and Key must be provided, it can be automatically obtained from Cloud.
+		</description>
+
+		<channels>
+			<channel id="power" typeId="power"/>
+			<channel id="target-temperature" typeId="target-temperature"/>
+			<channel id="operational-mode" typeId="operational-mode"/>
+			<channel id="fan-speed" typeId="fan-speed"/>
+			<channel id="swing-mode" typeId="swing-mode"/>
+			<channel id="eco-mode" typeId="eco-mode"/>
+			<channel id="turbo-mode" typeId="turbo-mode"/>
+			<channel id="indoor-temperature" typeId="indoor-temperature"/>
+			<channel id="outdoor-temperature" typeId="outdoor-temperature"/>
+			<channel id="sleep-function" typeId="sleep-function"/>
+			<channel id="temperature-unit" typeId="temperature-unit"/>
+
+			<channel id="on-timer" typeId="on-timer"/>
+			<channel id="off-timer" typeId="off-timer"/>
+			<channel id="appliance-error" typeId="appliance-error"/>
+			<channel id="auxiliary-heat" typeId="auxiliary-heat"/>
+			<channel id="humidity" typeId="humidity"/>
+			<channel id="screen-display" typeId="screen-display"/>
+			<channel id="alternate-target-temperature" typeId="alternate-target-temperature"/>
+			<channel id="dropped-commands" typeId="dropped-commands"/>
+		</channels>
+
+		<representation-property>ipAddress</representation-property>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>ipAddress</context>
+				<label>IP Address</label>
+				<description>IP Address of the device.</description>
+			</parameter>
+			<parameter name="ipPort" type="text" required="true">
+				<context>ipPort</context>
+				<label>IP Port</label>
+				<description>IP port of the device (for V2: 6444).</description>
+				<default>6444</default>
+			</parameter>
+			<parameter name="deviceId" type="text" required="true">
+				<context>deviceId</context>
+				<label>Device ID</label>
+				<description>ID of the device. Leave 0 to do ID discovery.</description>
+				<default>0</default>
+			</parameter>
+			<parameter name="cloud" type="text" required="false">
+				<context>cloud</context>
+				<label>Cloud Provider</label>
+				<description>Cloud Provider name for email and password.</description>
+				<options>
+					<option value=""></option>
```

Comment:
I thought I responded to this.  There a couple of concerns, one is that "" is the default to decipher the discovery message.  And to get the AC on the LAN one of the apps needed to be used, so a V2 user could either leave blank after discovery or add one of the three defined options.  I think it would be confusing to have a default.

Reply:
Not sure if i understand it. But the value can still remain empty, but the option needs a label so that it works nice in the UI. 
Example:
`<option value="">Not set (V2) or auto detect</option>`

Reply:
Ok misunderstood.  Just the description